src/sentry/static/sentry/app/components/charts/percentageAreaChart.jsx

```diff
@@ -64,11 +64,14 @@ export default class PercentageAreaChart extends React.Component {
           // Make sure tooltip is inside of chart (because of overflow: hidden)
           confine: true,
           formatter: seriesParams => {
+            // `seriesParams` can be an array or an object :/
```

Comment:
isn't it always an array in the line below?

Reply:
Right, we coerce `seriesParams` to always be an array but we store it into a new var